app/views/directives/resource-service-bindings.html

```diff
@@ -1,9 +1,10 @@
 <div class="mar-bottom-xl" ng-if="$ctrl.showBindings">
-  <h3>Service Bindings</h3>
+  <h3>Bindings</h3>
   <overview-service-binding
```

Comment:
I'm not sure we should reuse the `overview-service-binding` component here. If we do, it should be renamed.

Reply:
Sure thing, np renaming it. It just happens to be spread out over several files, little worried I'd end up with more merge conflicts =). However since it sounds like we're good with it I'll go ahead and do the search and replace.